### PR TITLE
{math}[foss/2020b] gmsh v4.8.4 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
aion-0027 - Linux RHEL 8.3, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/2beb7f7bfa9edafa964bfb4ec56185e3 for a full test report.

**Comment by Falkor** [comment]
I don't know if adding the dependant EB from https://github.com/easybuilders/easybuild-easyconfigs/pull/13904 was pertinent -- it was to ensure you are able to reproduce the build from this sole PR. Feel free to remove `SLEPc-3.14.2-foss-2020b.eb` from this PR to keep both management separated.

**Comment by boegel** [comment]
@Falkor It's better not to merge PRs together like you did, but instead just mark that another PR is required to be merged first by referring to it in the PR description, like:

requires #13904

We can re-trigger the tests once the other PR has been merged.

File: easybuild/easyconfigs/g/gmsh/gmsh-4.8.4-foss-2020b-Python-3.8.6.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '4.8.4'
+versionsuffix = '-Python-%(pyver)s'
```
Comment (boegel) [inline easybuild/easyconfigs/g/gmsh/gmsh-4.8.4-foss-2020b-Python-3.8.6.eb:5]: @Falkor Please drop the `versionsuffix`, no longer needed for Python 3 in the `2020b` generation of easyconfigs

**Review by boegel — CHANGES_REQUESTED** [review]
Please also sync with `develop` branch now that #13904 is merged (you can use `--sync-pr-with-develop 13905`)

File: easybuild/easyconfigs/g/gmsh/gmsh-4.8.4-foss-2020b-Python-3.8.6.eb (hunk 1 above)
Comment (Falkor) [reply easybuild/easyconfigs/g/gmsh/gmsh-4.8.4-foss-2020b-Python-3.8.6.eb:5]: @boegel Thanks for the comments, I corrected accordingly. Shall I rename also the EB to `gmsh-4.8.4-foss-2020b.eb` ?

Comment (Falkor) [reply easybuild/easyconfigs/g/gmsh/gmsh-4.8.4-foss-2020b-Python-3.8.6.eb:5]: my bad, big mess and mistake in the EB file to correct -- sry about that, I'll correct it asap

File: easybuild/easyconfigs/g/googletest/googletest-1.10.0-GCCcore-9.3.0.eb [hunk 2]
```diff
@@ -16,13 +16,11 @@ builddependencies = [
     ('binutils', '2.34'),
     ('CMake', '3.16.4'),
 ]
-# build twice, once for static, once for shared libraries
-configopts = ['', ' -DBUILD_SHARED_LIBS=ON ']
```
Comment (boegel) [inline easybuild/easyconfigs/g/googletest/googletest-1.10.0-GCCcore-9.3.0.eb:20]: @Falkor This looks like an unintended change that slipped in somehow?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/googletest/googletest-1.10.0-GCCcore-9.3.0.eb (hunk 2 above)
Comment (Falkor) [reply easybuild/easyconfigs/g/googletest/googletest-1.10.0-GCCcore-9.3.0.eb:20]: No, I have it also from official EB 4.4.1 ? 

```console 
$ eb -S googletest-1.10.0
CFGS1=/path/to/software/EasyBuild/4.4.1/easybuild/easyconfigs/g/googletest
CFGS2=/path/to/github.com/ULHPC/easybuild-easyconfigs/easybuild/easyconfigs/g/googletest
 * $CFGS1/googletest-1.10.0-GCCcore-8.3.0.eb
 * $CFGS1/googletest-1.10.0-GCCcore-9.3.0.eb
 * $CFGS1/googletest-1.10.0-GCCcore-10.2.0.eb
 * $CFGS2/googletest-1.10.0-GCCcore-8.3.0.eb
 * $CFGS2/googletest-1.10.0-GCCcore-9.3.0.eb
 * $CFGS2/googletest-1.10.0-GCCcore-10.2.0.eb
```
Then: 
```bash 
# force check from 4.4.1 
CFGS1=/path/to/software/EasyBuild/4.4.1/easybuild/easyconfigs/g/googletest
$ cat $CFGS1/g/googletest/googletest-1.10.0-GCCcore-9.3.0.eb
easyblock = 'CMakeMake'

name = 'googletest'
version = '1.10.0'

homepage = 'https://github.com/google/googletest'
description = "Google's framework for writing C++ tests on a variety of platforms"

toolchain = {'name': 'GCCcore', 'version': '9.3.0'}

source_urls = ['https://github.com/google/googletest/archive/']
sources = ['release-%(version)s.tar.gz']
checksums = ['9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb']

builddependencies = [
    ('binutils', '2.34'),
    ('CMake', '3.16.4'),
]
# build twice, once for static, once for shared libraries
configopts = ['', ' -DBUILD_SHARED_LIBS=ON ']
separate_build_dir = True

sanity_check_paths = {
    'files': ['lib/lib%s.%s' % (local_lib, local_ext) for local_lib in ['gmock', 'gmock_main', 'gtest', 'gtest_main']
              for local_ext in ['a', SHLIB_EXT]],
    'dirs': ['include/gmock', 'include/gtest'],
}

moduleclass = 'tools'


```

**Comment by Falkor** [comment]
It was kept in [`googletest-1.10.0-GCCcore-10.2.0.eb`](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/googletest/googletest-1.10.0-GCCcore-10.2.0.eb). I guess the requested change can be dropped ?

**Comment by Falkor** [comment]
@boegel you confirm?

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

@Falkor Sorry for not getting back to this...

We also have easyconfigs for more recent versions of `gmsh` already

**Pull request closed without merging**